src/pages/Home.tsx

```diff
@@ -1,5 +1,13 @@
 import React from 'react';
+import Footer from '../components/shared/footer/Footer';
 
 export function Home() {
-  return <div>Home</div>;
+  return (
+    <>
+      <div className="flex flex-col min-h-screen">Home</div>
+      <Footer />
+    </>
+  );
```

Comment:
Great work on the footer, but I don't believe this is where the footer should sit.

It should be common to all pages.

You can try and go to the `Layout` file, and add the footer beneath the children.